### PR TITLE
How to edit Hub using GitHub Web Editor

### PR DESCRIPTION
**Pull request by tnabtaf**
Incorporates new doc on how to edit the Galaxy Hub using the GitHub web editor interface.  This splits the existing content (mostly about editing a local copy, and mostly unchanged in this PR) in the README.md and puts the editing bits in the CONTRIBUTING.md file.

### CODE REVIEW

File: README.md.bak [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Galaxy Community Hub [![Build Status](https://jenkins.galaxyproject.org/buildStatus/icon?job=galaxy-hub)](https://jenkins.galaxyproject.org/job/galaxy-hub/)
```
Comment (martenson) [inline README.md.bak]: you probably do not want to commit this

File: CONTRIBUTING.md.bak [hunk 2]
```diff
@@ -0,0 +1,655 @@
+# Galaxy Community Hub [![Build Status](https://jenkins.galaxyproject.org/buildStatus/icon?job=galaxy-hub)](https://jenkins.galaxyproject.org/job/galaxy-hub/)
```
Comment (martenson) [inline CONTRIBUTING.md.bak]: You probably do not want to commit this backup file?

Comment (tnabtaf) [reply CONTRIBUTING.md.bak]: Probably not.  Sorry, first time I used aspell, and I was not paying attention.  I'll drop them.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,655 @@
...
+
+1. Until you get the hang of this ( *see What happens next and becoming an Editor* below) all of your edits will be reviewed by an experienced editor before they go to the web site.
+2. There are guidelines below, but don't worry about them too much when you start.  Just focus on the content you are fixing or adding. If you really want to "do the right thing" focus on the basic guidelines first, and then expand out as you need to.
+3. **Be bold.** We mean it.
+
+### You mentioned guidelines?
+
+Guidelines are distributed throughout this document.  Each is tagged with one of these icons:
+
+- :hatching_chick:  - A basic guideline, represented by a hatching chick.  Try to follow these from the start.
```
Comment (martenson) [inline CONTRIBUTING.md]: you still have both - the bullet list of chicks and the table of chicks

Comment (tnabtaf) [reply CONTRIBUTING.md]: Which representation is better?  

I like the list, but am willing to convert everything to tables if the consensus is otherwise.

Comment (martenson) [reply CONTRIBUTING.md]: +1 on the bullets

Comment (tnabtaf) [reply CONTRIBUTING.md]: Dropping replacing tables.

**Comment by tnabtaf** [comment]
Ping.  I just brought the branch up to date with master.  Any chance this can be approved?

**Comment by martenson** [comment]
@tnabtaf not just approved, merged!

**Pull request merged**